src/AppInstallerCommonCore/UserSettings.cpp

```diff
@@ -229,9 +229,32 @@ namespace AppInstaller::Settings
         WINGET_VALIDATE_PASS_THROUGH(EFExperimentalArg)
         WINGET_VALIDATE_PASS_THROUGH(EFDependencies)
         WINGET_VALIDATE_PASS_THROUGH(TelemetryDisable)
+        WINGET_VALIDATE_PASS_THROUGH(AddToPathEnvironment)
         WINGET_VALIDATE_PASS_THROUGH(EFDirectMSI)
         WINGET_VALIDATE_PASS_THROUGH(EnableSelfInitiatedMinidump)
 
+        WINGET_VALIDATE_SIGNATURE(InstallRoot)
+        {
+            const std::filesystem::path installRoot = value;
+
+            if (!std::filesystem::exists(installRoot))
+            {
+                try
+                {
+                    if (!std::filesystem::create_directories(installRoot))

```

Comment:
I don't think this ends well if the first time is via an elevated context and subsequent attempts are not.  We need special code to properly create directories that can be modified by the medium IL processes that will eventually follow.